system/sqlstore.go

```diff
@@ -71,7 +71,7 @@ func (s *SQLStore) New(r *http.Request, name string) (*sessions.Session, error)
 	err = securecookie.DecodeMulti(name, c.Value, &session.ID, s.codecs...)
 	if err != nil {
 		// these are not the sessions you are looking for
-		log.Infof("sqlstore: New: unable to decode cookie: %v", err)
+		log.Debugf("sqlstore: New: unable to decode cookie: %v", err)
```

Comment:
Why is the log level changing here?

Reply:
At the time I made the sql store, I saw this log quite a bit.  When an operator first updates from whatever to a version using the sql store, depending on the number of users, this log may flood their stdout.  It's not really a problem, it's finding the old type of cookie in the browser, that we can't decode in this fashion, which soon gets replaced. That's why I don't think this log is useful to anyone that's not debugging.  I have not seen the log since then.
Is it better to stay info? It might even be changed to warn once everyone has updated...

Reply:
Thanks for the explanation. I think that is a decent reason to change the log level to Debug.